aks_rcs_client/aks_rcs_client.py

```diff
@@ -0,0 +1,71 @@
+#usr/bin/env python3
+import rclpy
+import paho.mqtt.client as mqtt
+from rclpy.node import Node
+from std_msgs.msg import String
+
+class aks_rcs_client(Node):
+    def __init__(self):
+        super().__init__("aks_rcs_client")
+
+        self.declare_parameters(
+            namespace='',
+            parameters=[
+                ('mqtt_host', "127.0.0.1"),                     # ADDRESS TO MQTT BROKER
+                ('mqtt_port', 1883),                            # PORT TO MQTT BROKER
+                ('team_id', "AMP"),                             # USED FOR CLIENT NAME AND MQTT KART STATE TOPIC
```

Comment:
```suggestion
                ('team_id', "TEAM_NAME"),                             # USED FOR CLIENT NAME AND MQTT KART STATE TOPIC
```

Use arbitrary name since it will not only be used by us!